core/domain/email_manager.py

```diff
@@ -208,6 +210,35 @@ def log_new_error(*args, **kwargs):
     []
 )
 
+NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_INFO = {
+    'email_body_template': (
+        'Hi %s,<br><br>'
+        'There are new review opportunities that we think you might be '
+        'interested in on the '
+        '<a href="https://www.oppia.org/contributor-dashboard/">Contributor '
```

Comment:
Just to confirm, the contributor dashboard isn't yet available in production, correct? (This URL does not work, so I'm just double checking that it's formatted correctly since I can't manually verify it).